t/node/healthcheck-https.t

```diff
@@ -0,0 +1,345 @@
+        lua_ssl_trusted_certificate ../../certs/mtls_ca.crt;
+        content_by_lua_block {
+            local t = require("lib.test_admin")
+            local core = require("apisix.core")
+            local cert = t.read_file("t/certs/mtls_client.crt")
+            local key =  t.read_file("t/certs/mtls_client.key")
+            local data = {
+                uri = "/ping",
+                upstream = {
+                    scheme = "https",
+                    nodes = {
+                        ["127.0.0.1:8765"] = 1,
+                        ["127.0.0.1:8767"] = 1
+                    },
+                    tls = {
+                        client_cert = cert,
+                        client_key = key
+                    },
+                    retries = 2,
+                    checks = {
+                        active = {
+                            type = "https",
+                            http_path = "/healthz",
+                            https_verify_certificate = false,
+                            healthy = {
+                                interval = 1,
+                                successes = 1
+                            },
+                            unhealthy = {
+                                interval = 1,
+                                http_failures = 1
+                            },
+                        }
+                    }
+                }
+            }
+            local code, body = t.test('/apisix/admin/routes/1',
+                ngx.HTTP_PUT, core.json.encode(data))
+            if code >= 300 then
+                ngx.status = code
+                ngx.say(body)
+                return
+            end
+
+            local http = require("resty.http")
+            local httpc = http.new()
+            local uri = "http://127.0.0.1:" .. ngx.var.server_port .. "/ping"
+            local _, _ = httpc:request_uri(uri, {method = "GET", keepalive = false})
+            ngx.sleep(0.5)
+
+            local healthcheck_uri = "http://127.0.0.1:" .. ngx.var.server_port .. "/v1/healthcheck/routes/1"
+            local httpc = http.new()
+            local res, _ = httpc:request_uri(healthcheck_uri, {method = "GET", keepalive = false})
+            local json_data = core.json.decode(res.body)
+            assert(json_data.type == "https")
+            assert(#json_data.nodes == 2)
+
+            local function check_node_health(port, status)
+                for _, node in ipairs(json_data.nodes) do
+                    if node.port == port and node.status == status then
+                        return true
+                    end
+                end
+                return false
+            end
+
+            assert(check_node_health(8765, "healthy"), "Port 8765 is not healthy")
+            assert(check_node_health(8767, "healthy"), "Port 8767 is not healthy")
+        }
+    }
+--- request
+GET /t
+--- error_code: 200
+
+
+
+=== TEST 1: https health check (one healthy node, one unhealthy node)
```

Comment:
```suggestion
=== TEST 2: https health check (one healthy node, one unhealthy node)
```

Reply:
please use `utils/reindex *.t` to reindex the file

Reply:
why CI not found this ?

![image](https://github.com/apache/apisix/assets/2706161/0ee6517b-2d2a-49f4-bbff-75c2c24f4c63)